### PR TITLE
Web portal: Add a new page to show hardware utilization detail for each node.

### PR DESCRIPTION
**Pull request by hwuu**
image

### CODE REVIEW

File: webportal/src/app/cluster-view/hardware/hardware-detail.component.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const hardwareDetailComponent = require('./hardware-detail.component.ejs');
+const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
+const webportalConfig = require('../../config/webportal.config.json');
+const url = require('url');
+
+//
+
+const resizeContentWrapper = () => {
+  $('#content-wrapper').css({'height': $(window).height() + 'px'});
+  $('#content-iframe').css('height', (($(window).height() - 120)) + 'px');
```
Comment (sterowang) [inline webportal/src/app/cluster-view/hardware/hardware-detail.component.js]: What does the magic "120" means?

Comment (hwuu) [reply webportal/src/app/cluster-view/hardware/hardware-detail.component.js]: It is the number of pixels from the top of the page to the top of the iframe -- I haven't found a solution to make the iframe automatically fit into the space.

Comment (sterowang) [reply webportal/src/app/cluster-view/hardware/hardware-detail.component.js]: Not looks perfect, but let's move on first.

Comment (hwuu) [reply webportal/src/app/cluster-view/hardware/hardware-detail.component.js]: Agree. Will find a way eventually.

**Pull request merged**